tests/cloudpickle_test.py

```diff
@@ -1640,6 +1640,16 @@ def g(x):
 
         self.assertEqual(f2.__annotations__, f.__annotations__)
 
+    @unittest.skipIf(sys.version_info < (3, 7),
+                     """This syntax won't work on py2 and pickling annotations
+                     isn't supported for py37 and below.""")
```

Comment:
Just use "single" double quotes here: there are parens around, so it's fine to not use explicit concatenation:

```python
    @unittest.skipIf(sys.version_info < (3, 7),
                     "This syntax won't work on py2 and pickling annotations "
                     "isn't supported for py37 and below.")
...
```